app/services/affiliate-slots.js

```diff
@@ -123,10 +124,39 @@ export default Service.extend({
   logger: service(),
   geo: service(),
   wikiVariables: service(),
+  currentUser: service(),
 
   currentWikiId: readOnly('wikiVariables.id'),
   currentVertical: readOnly('wikiVariables.vertical'),
   currentCountry: readOnly('geo.country'),
+  currentUserId: readOnly('currentUser.userId'),
+
+  _updateUnitLink(unit, pageId = 'search') {
+    if (!unit || unit.campaign !== 'ddb') {
+      return unit;
+    }
+
+    const beaconId = Cookies.get('wikia_beacon_id');
+    const session = Cookies.get('wikia_session_id');
+    const userId = this.currentUserId ? this.currentUserId : 'null'; // make sure we actually send null
+    const wikiId = this.currentWikiId;
+
+    // fandom_slot_id will be added later
+    const questionMarkOrAmpersan = (unit.link.indexOf('?') > -1) ? '&' : '?';
+    unit.link = `${unit.link}${questionMarkOrAmpersan}fandom_session_id=${session}&fandom_user_id=${userId}&fandom_campaign_id=${unit.category}&fandom_community_id=${wikiId}fandom_page_id=${pageId}&fandom_beacon_id=${beaconId}`;
+    return unit;
+  },
+
+  _createDDBLink(unit, pageId = 'search') {
+    // const beaconId = Cookies.get('wikia_beacon_id');
```

Comment:
Should this be commented out here?

Reply:
Good catch I meant to remove this entire block : https://github.com/Wikia/mobile-wiki/pull/1937/commits/ddec7df2ecfa854a7b01847845ecc212af80ae9c